group_vars/rhcs.yml.sample

```diff
@@ -702,7 +702,7 @@ grafana_container_image: registry.redhat.io/rhceph/rhceph-5-dashboard-rhel8:5
 #grafana_allow_embedding: True
 #grafana_port: 3000
 #grafana_conf_overrides: {}
-prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.1
+prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.6
```

Comment:
We need to use this image registry.redhat.io/openshift4/ose-prometheus:v4.6 for prometheus with version 2.22.2